sfreeley/front-end-capstone
# Sf modal style

# Description
Fixes #4 
1. Fixed search bar issue where searching and giving results correct with onChange, but then after backspacing or clearing the search input field it will show all results in the search array instead of all the results being removed (was asynchronous issue)
2. Fixed save to medication history issue where if user searches for a medication card or medication history card they are not able to send it med hx from the search result; 
3. Fixed edit issue where if user searches on home page/details page for a medication card or medication history card they are not able to edit with modal
4. Fixed issue where user cannot completely logout from newly added resources page
5. Also added edit functionality from search bar in med history page (modal should appear now)
6. Click logout from every page and applications tab with session storage in dev tools should erase user completely and bring back to login page

## Type of change

# Testing Instructions
1. Pull down from sf-modal-style branch
2. search "crestor" from homepage- functionality should include: edit, expand, delete, save to med hx/medlist (whichever one is applicable)
3. search functionality should clear all results after fully backspacing or deleting the input from search field
4. Should be able to edit from details and hx page with modal 

# Checklist:
